cmd/nodecmd/create.go

```diff
@@ -76,6 +77,12 @@ var (
 	grafanaPkg           string
 	wizSubnet            string
 	publicHTTPPortAccess bool
+
+	bootstrapIDs  []string
+	bootstrapIPs  []string
+	genesisPath   string
+	upgradePath   string
+	useEtnaDevnet bool
```

Comment:
I think `useEtnaDevnet` should be put into the existing `NetworkFlags` rather than being treated differently here.

Reply:
etnadevnet is a shortcut for the devnet
we already have devnet here so I don't think we should

Reply:
good point.
etna devnet should be treated like fuji, the only difference is lack of prebuilt genesis/boostrapper info that is included into avalanchego pkg. For enta devnet we need to specify bootstrap config in avalanchego config. We just have it in constants to help user

Reply:
in my opinion it should be something like `custom`

Reply:
More generally, it seems like `network_options.go` currently contains logic for supported networks to connect to. I think it would be error prone and less organized to put other network option logic in different places. This way it's all contained in the same places so it's easier to reason about.